docs/framework/whats-new/index.md

```diff
@@ -164,44 +164,40 @@ The .NET Framework 4.6.2 includes new features in the following areas:
   
 -   [Debugging improvements](#Debug462)  
   
- For a list of new APIs added to the .NET Framework 4.6.2, see [.NET Framework 4.6.2 API Changes](https://github.com/Microsoft/dotnet/blob/master/releases/net462/dotnet462-api-changes.md) on GitHub. For a list of feature improvements and bug fixes in the .NET Framework 4.6.2, see [.NET Framework 4.6.2 API Changes](http://go.microsoft.com/fwlink/?LinkId=708778) on GitHub.  For additional information, see [Announcing .NET Framework 4.6.2](https://blogs.msdn.microsoft.com/dotnet/2016/08/02/announcing-net-framework-4-6-2/) in the .NET Blog.  
+ For a list of new APIs added to the [!INCLUDE[net_v462](../../../includes/net-v462-md.md)], see [.NET Framework 4.6.2 API Changes](https://github.com/Microsoft/dotnet/blob/master/releases/net462/dotnet462-api-changes.md) on GitHub. For a list of feature improvements and bug fixes in the [!INCLUDE[net_v462](../../../includes/net-v462-md.md)], see [.NET Framework 4.6.2 API Changes](http://go.microsoft.com/fwlink/?LinkId=708778) on GitHub.  For additional information, see [Announcing .NET Framework 4.6.2](https://blogs.msdn.microsoft.com/dotnet/2016/08/02/announcing-net-framework-4-6-2/) in the .NET Blog.  
   
 <a name="ASPNET462"></a>   
 ### ASP.NET  
- In the .NET Framework 4.6.2, ASP.NET includes the following enhancements:  
+ In the [!INCLUDE[net_v462](../../../includes/net-v462-md.md)], ASP.NET includes the following enhancements:  
   
  **Improved support for localized error messages in data annotation validators**  
- Data annotation validators enable you to perform validation by adding one or more attributes to a class property. The attribute's [ValidationAttribute.ErrorMessage](assetId:///P:System.ComponentModel.DataAnnotations.ValidationAttribute.ErrorMessage?qualifyHint=True&autoUpgrade=True) element defines the text of the error message if validation fails. Starting with the .NET Framework 4.6.2, ASP.NET makes it easy to localize error messages. Error messages will be localized if:  
+ Data annotation validators enable you to perform validation by adding one or more attributes to a class property. The attribute's <xref:System.ComponentModel.DataAnnotations.ValidationAttribute.ErrorMessage%2A?displayProperty=fullName> element defines the text of the error message if validation fails. Starting with the [!INCLUDE[net_v462](../../../includes/net-v462-md.md)], ASP.NET makes it easy to localize error messages. Error messages will be localized if:  
```

Comment:
No, the * means the link is going to the overload page instead of going to the specific overload. It was the default behavior in our old tool.